### PR TITLE
rename issuer name on 1095A forms as approved by ME

### PR DESCRIPTION
**Pull request by saipraveen18**
https://www.pivotaltracker.com/story/show/186528262

### CODE REVIEW

**Review by saikumar9 — APPROVED** [review]
lgtm!

File: app/models/irs_yearly_pdf_report.rb [hunk 1]
```diff
@@ -219,6 +219,15 @@ def fill_subscriber_details
   end
   # rubocop:enable Metrics/AbcSize
 
+  def fetch_insurance_provider_title(title)
```
Comment (saikumar9) [inline app/models/irs_yearly_pdf_report.rb:222]: @saipraveen18 This code should only work for the client ME and not for DC. We need to use the resource registry or the existing implemented pattern to ensure there is different behavior b/w clients.

**Review by saikumar9 — CHANGES_REQUESTED** [review]

**Review by saikumar9 — APPROVED** [review]
lgtm!

**Review by vkghub — APPROVED** [review]
LGTM

**Pull request merged**